### PR TITLE
Create intro bots twitter

### PR DESCRIPTION
**Pull request by spapastamkou**
*New FR translated lesson*

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by spapastamkou** [comment]
Tout a l'air de fonctionner, @programminghistorian/french-team pour vérifier et approuver la pull request. Merci!

File: fr/lecons/intro-aux-bots-twitter.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+ 
+Mais je crois aussi qu’il y a de la place en histoire et dans les humanités numériques de façon plus large pour un travail créatif, expressif, voire artistique. Les historiens et les historiennes qui connaissent la programmation peuvent profiter des possibilités offertes par les médias numériques pour monter des créations, autrement impossibles à réalise,r pour nous émouvoir, nous inspirer, nous interpeller. Il y a de la place pour de la satire, il y a de la place pour commenter. Comme Mark Sample, je crois qu’il y a besoin de &laquo; [bots de conviction](https://medium.com/@samplereality/a-protest-bot-is-a-bot-so-specific-you-cant-mistake-it-for-bullshit-90fe10b7fbaa)&raquo;.
```
Comment (mariechristineb) [inline fr/lecons/intro-aux-bots-twitter.md]: "autrement impossibles à réalise,r"

virgule intempestive au milieu de 'réaliser'

File: fr/lecons/intro-aux-bots-twitter.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+Les modificateurs `.capitalize` et `.s` sont ajoutés à l'intérieur du `#` du symbole qu'ils sont destinés à modifier. D'autres modificateurs sont `.ed`pour le passé, et `.a` pour a/an. Il y en a peut-être plus, car Tracery est en cours de développement.
+
+*N.D.L.R.: Cet exemple est adapté à l'anglais, nous avons néanmoins opté pour le présenter tel quel afin de démontrer les possibilités de Tracery, dont les fonctionnalités peuvent être développées selon les besoins d'une langue. Pour l'instant, les fonctionnalités les plus subtiles sont développées dans des contextes anglophones, elles sont donc plus adaptées aux règles de grammaire et de syntaxe de l'anglais.*  
+
+## Utiliser des emoji
+
+Les emojis peuvent être utilisés avec beaucoup d'efficacité dans des bots Twitter. Vous pouvez copier et coller des emojis directement dans l'éditeur Cheap Bots Done Quick, en les plaçant chacun entre guillemets comme toute autre valeur qui vous sert de règle. Utilisez [cette liste](http://unicode.org/emoji/charts/full-emoji-list.html) pour repérer les emojis que vous souhaitez utiliser, en veillant à ler copier/coller depuis la colonne Twitter pour vous assurer qu'ils vont bien s'afficher.
```
Comment (mariechristineb) [inline fr/lecons/intro-aux-bots-twitter.md]: 'en veillant à ler copier/coller'

ler -> les

File: fr/lecons/intro-aux-bots-twitter.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+
+Cette fonctionnalité ne serait probablement pas très utile dans le cas d'un bot Twitter, qui par définition produit du texte court. Mais lorsqu'il s'agit de générer du texte plus long, par exemple une histoire ou un poème, cette fonctionnalité peutt être employée pour que Tracery retienne la première fois qu'une règle particulière a été affectée à un symbole. Ainsi, nous pourrions faire en sorte que la même valeur soit toujours utilisée lors de chaque appel suivant de `creature`. C'est ce que Tracery appelle une « action ». La convention est #[uneAction]unSymbole#. Toutefois, cette fonctionnalité reste en cours de développement et peut prêter à confusion. Vous pouvez néanmoins voir comment elle fonctionne, si vous copiez-collez le JSON ci-dessous dans cet éditeur Tracery de Beau Gunderson : [https://beaugunderson.com/tracery-writer/](https://beaugunderson.com/tracery-writer/) (veiller à sélectionner et à supprimer le texte qui s'affiche par défaut). L'éditeur Tracery utilisé précédemment ne gère pas très bien la sauvegarde des données, donc cette alternative-ci constitue un meilleur outil pour nos besoins actuels.
```
Comment (mariechristineb) [inline fr/lecons/intro-aux-bots-twitter.md]: 'fonctionnalité peutt être employée'

peutt -> peut

**Review by mariechristineb — APPROVED** [review]

**Pull request merged**